apps/wallet/src/components/CheckedInput.tsx

```diff
@@ -1,31 +1,33 @@
 import type { FC } from 'react';
-import { useState } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import type { InputProps } from '@walless/gui';
 import { Input } from '@walless/gui';
 
 type Props = {
-	checkFunction: (value?: string) => string | undefined;
+	errorText?: string;
+	onFocusOut?: (value?: string) => void;
 } & InputProps;
 
-const CheckedInput: FC<Props> = ({ value, checkFunction, ...inputProps }) => {
-	const [errorText, setErrorText] = useState<string>();
-
-	const handlerBlur = () => {
-		setErrorText(checkFunction(value));
-	};
-
+const CheckedInput: FC<Props> = ({
+	value,
+	errorText,
+	onFocusOut,
+	...inputProps
+}) => {
```

Comment:
when I detached this component, I thought that the `errorText` was just used in this component, so keeping the state here might be good at rendering performance.